pkg/controller/nmstate/nmstate_controller.go

```diff
@@ -112,6 +114,17 @@ func (r *ReconcileNMstate) Reconcile(request reconcile.Request) (reconcile.Resul
 		return reconcile.Result{}, err
 	}
 
+	// Wait for the handler to be fully up
+	instance.Status.LastUpdated = &metav1.Time{Time: time.Now()}
+	err = r.client.Status().Update(context.Background(), instance)
+	if err != nil {
+		// Request object not found, could have been deleted after reconcile request.
+		if !apierrors.IsNotFound(err) {
+			errors.Wrap(err, "Request object not found, could have been deleted after reconcile request")
```

Comment:
Two notes:
- This is "no" IsNotFound so the error message is wrong
- Since this is kind of "cosmetics" let's just log and error for Status updates and not fail `Reconcile`so we don't re-apply manifests.